docs/examples/contrib/piccolo/piccolo_conf.py

```diff
@@ -1,3 +1,3 @@
 from piccolo.engine.sqlite import SQLiteEngine
 
-DB = SQLiteEngine()
+DB = SQLiteEngine(path=":memory:?cache=shared")
```

Comment:
I'm sorry, but I don't know how to do it because we use that exact `path` to create the `Task` table in `app.py` and for `app` execution in tests. `SQLiteEngine` is specified in `piccolo_conf.py` and engine are what execute the SQL and it's important that each `Table` class knows which engine to use. Usually looks like this `DB = SQLiteEngine()` (when the default db is `piccolo.sqlite`) or `SQLiteEngine(path="test.sqlite")` if you want to specify db name. If you know another way, I would be grateful if you could help me.

Reply:
The only way I can do this is to set the default `SQliteEngine` (not use `path=":memory:?cache=shared"`) and then delete created `piccolo.sqlite` with `Path(DB.path).unlink()`. Would that be better?

Reply:
Maybe something like this in `tests/examples/test_contrib/test_piccolo/test_piccolo_crud.py`? @provinzkraut would know for sure.. just offering suggestion

```py
DB_TEST = SQLiteEngine(path=":memory:?cache=shared")

@pytest.fixture(autouse=True, scope="module")
def create_test_data():
    with patch('docs.example.piccolo.DB', new=DB_TEST):
    ...
```

Reply:
@JacobCoffee Thanks for your suggestion, but unfortunately it doesn't work because when we call `Task.create_table(if_not_exists=True).run_sync()` in the fixture, we use the engine (with connection string from `piccolo_conf`) from here
https://github.com/litestar-org/litestar/blob/ea6f1b91d3b91e013ef7dec5a209d41b42fa9243/docs/examples/contrib/piccolo/app.py#L13-L16
I apologize, but I don't know how to solve it differently than I wrote in this PR, because
`Task.alter().drop_table().run_sync()` and `Path(DB.path).unlink()` guarantee that we always have a fresh database for tests on each run.

Reply:
That's why I suggesting patching the `DB.path` directly, since I expected this to be taken into account when issuing further commands.

I guess since this is quite complicated to do in piccolo we'll just leave it as is.
Maybe I'll open up an issue there and ask for advice.